pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -86,7 +87,7 @@ func New(assetsDir string, profile *performancev1alpha1.PerformanceProfile) (*ma
 	}
 
 	mc.Spec.Config = *ignitionConfig
-	mc.Spec.KernelArguments = getKernelArgs(profile.Spec.HugePages, profile.Spec.CPU.Isolated)
+	mc.Spec.KernelArguments = getKernelArgs(profile.Spec.HugePages, profile.Spec.CPU.Isolated,profile.Spec.CPU.NonIsolated)
```

Comment:
@davidvossel get rid of `NonIsolated` CPU's, so probably you will need to adapt your work to it

Reply:
@cynepco3hahue  well whatever gets merged first will dictated the adjustments